sympy/physics/continuum_mechanics/beam.py

```diff
@@ -2234,3 +2234,64 @@ def deflection(self):
         the three axes.
         """
         return self._deflection
+
+    def helper_function_for_shearplot(self, dir, subs=None):
+
+        shear_force = self.shear_force()
+
+        if dir == 'x':
+            dir_num = 0
+        elif dir == 'y':
+            dir_num = 1
+        elif dir == 'z':
+            dir_num = 2
+
+        if subs is None:
+            subs = {}
+
+        for sym in shear_force[dir_num].atoms(Symbol):
+            if sym == self.variable:
+                continue
+            if sym not in subs:
+                raise ValueError('Value of %s was not passed.' %sym)
+        if self.length in subs:
+            length = subs[self.length]
+        else:
+            length = self.length
+
+        P = plot(shear_force[dir_num].subs(subs), (self.variable, 0, length), show = False, title='Shear Force along %c direction'%dir,
+                xlabel=r'$\mathrm{%c}$'%dir, ylabel=r'$\mathrm{V[%c]}$'%dir, line_color='g')
```

Comment:
Better to use different `line_color` for different axis.

Reply:
Also, you can either directly return `plot(......)` function or choose a better variable name rather than `P`